app/controllers/users_controller.rb

```diff
@@ -8,7 +8,7 @@ def me
 
   def update
     data = user_params
-    missing_fields = fields.select { |f| data[f].nil? }
+    missing_fields = fields.reject { |f| data[f].present? }
```

Comment:
I guess this is okay for now. Maybe in the future we could do a whole validation process on the data in a different class (not here on the controller).